### PR TITLE
Add dimensions property to cellprofiler.object.Objects (resolves #2449)

### LINKED ISSUE #2449: Add dimensions property to cellprofiler.object.Objects
Adding a dimensions property to `cellprofiler.object.Objects` will simplify choosing whether to extract two-or-three dimensional features.

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: cellprofiler/object.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ def __init__(self):
         self.__parent_image = None
 
     @property
+    def dimensions(self):
+        return self.__parent_image.dimensions
```
Comment (mcquin) [inline cellprofiler/object.py]: What if `self.__parent_image` is `None`?

Comment (0x00b1) [inline cellprofiler/object.py]: Fixed. :)

File: cellprofiler/object.py [hunk 2]
```diff
@@ -40,6 +40,15 @@ def __init__(self):
         self.__parent_image = None
 
     @property
+    def dimensions(self):
+        if self.__parent_image:
+            return self.__parent_image.dimensions
+        else:
```
Comment (mcquin) [inline cellprofiler/object.py]: `else` isn't super necessary here

**Comment by mcquin** [comment]
@0x00b1 needs rebase

**Pull request merged**